README.md

```
@@ -349,6 +417,24 @@ $ oc extract secret/osac-aap-admin-password -n <project-name> --to -
 - Password: (from the previous step)
 
+### Create an AAP API Token for the OSAC Operator
+
+The OSAC operator requires an API token to communicate with AAP. The `scripts/create-aap-token.sh`
+script automates this by authenticating with the AAP gateway using the admin password and
+creating a write-scoped token. The automated setup script (`setup.sh`) calls this automatically.
+
+For manual deployments, run it after the AAP bootstrap job completes:
+
+```bash
+$ INSTALLER_NAMESPACE=<project-name> ./scripts/create-aap-token.sh
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)